ui/app/components/log-stream/index.hbs

```diff
@@ -1,34 +1 @@
-<div class="output-pane">
-  {{#if this.lines}}
-    <div class="output-scroll-y">
-      {{#unless this.isFollowingLogs}}
-        <button class="follow-logs" type="button" {{on "click" this.followLogs}}>
-          <Pds::Icon @type="arrow-down" class="icon" />
-          {{#if (gt this.badgeCount 0)}}
-            <div class="badge">{{this.badgeCount}}</div>
-          {{/if}}
-        </button>
-      {{/unless}}
-      <div class="output-scroll-x" {{did-insert this.followLogs}}>
-        {{#each this.lines as |line|}}
-          <div class="log-line">
-            <code>{{line}}</code>
-          </div>
-        {{/each}}
-        <div class="output-end"
-          {{in-viewport
-            onEnter=(fn (mut this.isFollowingLogs) true)
-            onExit=(fn (mut this.isFollowingLogs) false)
-          }}
-          {{did-update this.updateScroll this.lines.length}}
-        ></div>
-      </div>
-    </div>
-  {{else}}
-    <EmptyState>
-      <p>There are no logs to display for this app yet</p>
-      <p>Logs will be displayed when you run actions in the CLI such as <code class="inline">build</code>,
-        <code class="inline">deploy</code> or <code class="inline">release</code></p>
-    </EmptyState>
-  {{/if}}
-</div>
\ No newline at end of file
+<Terminal @terminal={{this.terminal}}/>
```

Comment:
What do you think about calling the component `<RenderTerminal>` rather than just `<Terminal>`? That might make it slightly more explicit that the component expects to be given a `Terminal` object.

```suggestion
<RenderTerminal @terminal={{this.terminal}}/>
```